app/assets/stylesheets/components/find/_call-out-box.scss

```diff
@@ -0,0 +1,13 @@
+.app-find-pink-callout {
+  border-left-color: #d43681;
+  background-color: #f3f2f1;
+}
+
+.app-find-custom-grid-padding {
```

Comment:
I am not the best at CSS but is there a format of those class names to be used on CSS?